packages/contracts-bedrock/deploy-config/devnetL1-template.json

```diff
@@ -49,6 +49,8 @@
   "faultGameAbsolutePrestate": "0x03c7ae758795765c6664a5d39bf63841c71ff191e9189522bad8ebff5d4eca98",
   "faultGameMaxDepth": 30,
   "faultGameMaxDuration": 1200,
```

Comment:
we will need to add these to the other files too, otherwise they will break

Reply:
I don't believe that's the case - they've been there this whole time?

Reply:
the new fields in the json i mean